core/src/main/java/dev/streamx/cli/run/RunCommand.java

```diff
@@ -70,6 +79,7 @@ public void run() {
       print("Starting DX Mesh...");
 
       this.runner.startMesh();
+      setupRootAuthToken();
 
       printSummary(this.runner, meshDefinition.result().path());
       Quarkus.waitForExit();
```

Comment:
Clean up properties after mesh shut-down.

If public/private keys are not reused, then also clean up these.

Reply:
There is no need to clear properties. Every run request overrides existing value anyway. That is why i introduced root token instead of just using auth token. As for keys we do not want to generate them every time. They are reused once created